### DISCUSSION TITLE
Show original path and deletion date of items in the trash

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#569] [Ideas]
Currently it can be found at ~/.local/share/Trash/info/
But it would be more convenient to see it in the trash.

**Comment by mtwebster** [comment]
What file manager are you using? In Nemo the default trash view is:
![image](https://github.com/user-attachments/assets/b6f60b7a-589d-4d84-9a50-9c8b9e1382b2)

**Comment by ghost** [comment]
@mtwebster I did not change the file manager. It is not visible here:
![Screenshot from 2024-08-12 16-11-11](https://github.com/user-attachments/assets/f689ff8f-1782-4fb2-83d8-d86e9f692e5b)
